### PR TITLE
Charts: update Candlestick demo description

### PR DESCRIPTION
**Pull request by vladaskorohodova**

### CODE REVIEW

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 1]
```diff
@@ -1 +1,10 @@
...
\ No newline at end of file
+A Candlestick [series type](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. You can specify the Candlestick series type for each object in the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/Candlestick/description.md]: I would change the first sentence to better convey that *a single point* in a Candlestick chart display data over the course of a day. Right now, it seems like the entire chart only displays data for one day. 

The rest of the text then can be a separate paragraph. 

```suggestion
A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. 

You can use the Candlestick type for an individual [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 2]
```diff
@@ -1 +1,10 @@
...
+
+Additionally, use the [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) property to declare a custom tooltip that shows prices. You can also enable the [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/).[workdaysOnly](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#workdaysOnly) property to display only the work days on the argument axis.
```
Comment (Abadzhev) [inline JSDemos/Demos/Charts/Candlestick/description.md]: In this paragraph, you just need to mention a few additional options. You try to make it look like a continuous narrative by adding transitional phrases like "Additionally" or "You can also". But there's really no flow to these statements - they are separate from each other. I'd rather just use a list with a short introduction: 

```suggestion
The following settings may help you make your chart more user-friendly:

- [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) - specifies a  custom tooltip template so you can display additional information, such as price values. 
- [argumentAxis](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/).[workdaysOnly](/Documentation/ApiReference/UI_Components/dxChart/Configuration/argumentAxis/#workdaysOnly) - displays only work days on the argument axis.
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 3]
```diff
@@ -1 +1,15 @@
...
+
+The following settings may help you make your chart more user-friendly:
+
+- [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) - specifies a  custom tooltip template so you can display additional information, such as price values.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
- [tooltip](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/).[contentTemplate](/Documentation/ApiReference/UI_Components/dxChart/Configuration/tooltip/#contentTemplate) - specifies a custom tooltip template so you can display additional information, such as price values. 
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 4]
```diff
@@ -1 +1,15 @@
-In&nbsp;this demo, the **candle stick** series displays variations in&nbsp;stock price throughout the course of&nbsp;a&nbsp;given day. Each point consists of&nbsp;a&nbsp;rectangle (body) whose bottom and topmost values correspond to&nbsp;the opening and closing price of&nbsp;a&nbsp;stock. A&nbsp;vertical line (shadow, wick or&nbsp;tail) displays the high and low price for the stock. If&nbsp;the stock closes higher than its opening price, no&nbsp;color fills its body and if&nbsp;the stock closes lower than its opening price, a&nbsp;color fill is&nbsp;applied.
\ No newline at end of file
+A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]: Let's begin by briefly explaining what a Candlestick is. Just one sentence is enough. Something about it being a technical tool that packs data for multiple time frames into single price bars. After that, you'll need a transition sentence that focuses on the workings of our demo. Next, you can go into the sentence you already wrote above ("A single point...").

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 5]
```diff
@@ -1 +1,15 @@
...
+
+You can use the Candlestick type for an individual [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
+
+In Candlestick series, each point consists of a rectangle (body) and a vertical line (shadow, wick or tail). The top and bottom values of a vertical line correspond to the highest and lowest prices of the stock. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for these prices.
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
In Candlestick series, each point consists of a rectangle (body) and a vertical line (shadow, wick, or tail). The top and bottom values of a vertical line correspond to the highest and lowest prices of the stock, respectively. Use the [highValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#highValueField) and [lowValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#lowValueField) properties to specify data source fields for these prices.
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 6]
```diff
@@ -1 +1,15 @@
...
+
+The bottom and topmost values of the point's body correspond to the opening and closing price of a stock. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties. If the stock closes higher than its opening price, no color fills the body and if the stock closes lower than its opening price, the Chart applies a color fill to the body.
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
The bottom and topmost values of the point's body correspond to the opening and closing price of a stock. To specify these prices, use the [openValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#openValueField) and [closeValueField](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#closeValueField) properties. If the stock closes higher than its opening price, no color fills the body; if the stock closes lower than its opening price, the Chart applies a color fill to the body.
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 7]
```diff
@@ -1 +1,15 @@
...
+
+To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To change color of other points, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[color](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/#color) property; you can also specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[innerColor](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#innerColor) property to fill these points' bodies with color. To switch between high, low, open or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
To reflect changes on the market, the Chart component compares the price of every point with the price of the previous point. The Chart paints the points whose price has decreased in a specific [reduction color](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#color). To change color of other points, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[color](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/#color) property; you can also specify the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[innerColor](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#innerColor) property to fill these points' bodies with color. To switch between high, low, open, or close prices of points as sources for data comparison, use the [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/).[reduction](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/).[level](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/reduction/#level) property.
```

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 8]
```diff
@@ -1 +1,15 @@
...
+
+The following settings may help you make your chart more user-friendly:
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
The following settings may help you make your chart more user-friendly:
```
So, something about this sentence is off for me. The idea is fine, but that word "may" bothers me. How about: "Apply the following settings to make your chart more user-friendly. " 
What do you think?

**Review by arminal — COMMENTED** [review]
Vlada, you write well. I made a few edits, but you write well. Don't forget to use the oxford comma though. In technical writing especially, we shouldn't forget it. :)

File: JSDemos/Demos/Charts/Candlestick/description.md (hunk 8 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Charts/Candlestick/description.md]: I think that your sentence is better, I will replace it.

File: JSDemos/Demos/Charts/Candlestick/description.md (hunk 4 above)
Comment (vladaskorohodova) [reply JSDemos/Demos/Charts/Candlestick/description.md]: So I agree that I need a beginning sentence here, but I decided to use a different approach. Please take a look

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 9]
```diff
@@ -1,4 +1,4 @@
-A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. 
+The Candlestick financial Chart is designed to analyze trading patterns over a short period of time. A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day. 
 
 You can use the Candlestick type for an individual [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/) or specify the type in the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/) object for all series in the Chart. Use the [commonSeriesSettings](/Documentation/ApiReference/UI_Components/dxChart/Configuration/commonSeriesSettings/).[candlestick](/Documentation/ApiReference/UI_Components/dxChart/Series_Types/CandleStickSeries/) object to configure properties of the Chart Candlestick series.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Charts/Candlestick/description.md:3]: Test

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 10]
```diff
@@ -1 +1,15 @@
-In&nbsp;this demo, the **candle stick** series displays variations in&nbsp;stock price throughout the course of&nbsp;a&nbsp;given day. Each point consists of&nbsp;a&nbsp;rectangle (body) whose bottom and topmost values correspond to&nbsp;the opening and closing price of&nbsp;a&nbsp;stock. A&nbsp;vertical line (shadow, wick or&nbsp;tail) displays the high and low price for the stock. If&nbsp;the stock closes higher than its opening price, no&nbsp;color fills its body and if&nbsp;the stock closes lower than its opening price, a&nbsp;color fill is&nbsp;applied.
\ No newline at end of file
+The Candlestick financial Chart is designed to analyze trading patterns over a short period of time. A single point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```
Comment (arminal) [inline JSDemos/Demos/Charts/Candlestick/description.md]: Chart shouldn't start with a capital letter in this case.
The chart is not designed to analyze, but rather communicate those trading patterns. Humans do the analysis. 
```suggestion
The Candlestick financial chart is designed to communicate trading patterns over a short period of time. A single point [point? but I see rectangles and lines, but no points] on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```

Comment (vladaskorohodova) [reply JSDemos/Demos/Charts/Candlestick/description.md]: I meant "data point" here, each point on a line in a [line chart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/Spline/React/Light/), each bar in a [bar chart](https://js.devexpress.com/Demos/WidgetsGallery/Demo/Charts/StandardBar/React/Light/), each candlestick in a candlestick chart - all these things are data points, like each data object is represented by a data point in a Chart. And when you combine related data points together, they become series. Should I say "data point" instead? We just use simply "point" in our docs when referring to data points.

Comment (arminal) [reply JSDemos/Demos/Charts/Candlestick/description.md]: Yes, definitely say data point. :)

File: JSDemos/Demos/Charts/Candlestick/description.md [hunk 11]
```diff
@@ -1 +1,15 @@
-In&nbsp;this demo, the **candle stick** series displays variations in&nbsp;stock price throughout the course of&nbsp;a&nbsp;given day. Each point consists of&nbsp;a&nbsp;rectangle (body) whose bottom and topmost values correspond to&nbsp;the opening and closing price of&nbsp;a&nbsp;stock. A&nbsp;vertical line (shadow, wick or&nbsp;tail) displays the high and low price for the stock. If&nbsp;the stock closes higher than its opening price, no&nbsp;color fills its body and if&nbsp;the stock closes lower than its opening price, a&nbsp;color fill is&nbsp;applied.
\ No newline at end of file
+The Candlestick financial chart is designed to communicate trading patterns over a short period of time. A single point [point? but I see rectangles and lines, but no points] on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```
Comment (vladaskorohodova) [inline JSDemos/Demos/Charts/Candlestick/description.md]:
```suggestion
The Candlestick financial chart is designed to communicate trading patterns over a short period of time. A single data point on a Candlestick [series](/Documentation/ApiReference/UI_Components/dxChart/Configuration/series/#type) displays variations in stock prices over the course of a day.
```

**Pull request merged**